run.sh

```diff
@@ -1,14 +1,9 @@
-#!/bin/bash
+#!/bin/sh
 
+# Import GPG public keys
 gpg --import /keys/*
 
-cron
-
-# LS_COLORS is set to nothing and for some strange reason crontabs are not allowed to contain such env vars
-unset LS_COLORS
-
-# Create crontab file
-env | cat - > /backup.cron
```

Comment:
LS_COLORS is not set in Alpine and the environment variables should be available to cron...aren't they?